.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: docker
+
+on:
+  push:
+    branches:    
+      - master
+
+env:
+  IMAGE_NAME: git-chglog
+  VERSION: edge
+  IMAGE_ID: ghcr.io/git-chglog/git-chglog
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Build go binary
+        run: go build -o $IMAGE_NAME
+      - name: Build docker image
+        run: docker build . --tag $IMAGE_NAME --label "runnumber=${GITHUB_RUN_ID}"
```

Comment:
These two lines are not needed, we can build and tag at the same time, down below.

Reply:
sure, done